drivers/gpio/gpio_stm32.c

```diff
@@ -201,7 +218,7 @@ static inline uint32_t gpio_stm32_pin_to_exti_line(int pin)
 
 static void gpio_stm32_set_exti_source(int port, int pin)
 {
-	uint32_t line = gpio_stm32_pin_to_exti_line(pin);
+	u32_t line = gpio_stm32_pin_to_exti_line(pin);
 
```

Comment:
code cleaning

Reply:
Indeed, not directly linked, but reported by CI.